include/storage/io.hpp

```diff
@@ -104,6 +104,13 @@ class FileReader
         }
     }
 
+    void ReadInto(std::string &target)
+    {
+        auto len = ReadElementCount32();
```

Comment:
Hm.. I would say 64 bit here for consistency. But this would also mean we require 8 bytes per string. These strings are osm strings, no? So it should be bound by 255 * 4 bytes (max. 4 bytes in utf-8 coded code point), and 16 bit should suffice?